docs/operator-manual/user-management/google.md

```diff
@@ -200,38 +200,42 @@ Go through the same steps as in [OpenID Connect using Dex](#openid-connect-using
                 defaultMode: 420
                 secretName: argocd-google-groups-json
 
-3. Edit `argocd-cm` and add the following `dex.config` to the data section, replacing `clientID` and `clientSecret` with the values you saved before, `adminEmail` with the address for the admin user you're going to impersonate, and editing `redirectURI` with your Argo CD domain (note that the `type` is now `google` instead of `oidc`):
-
-        dex.config: |
-          connectors:
-          - config:
-              redirectURI: https://argocd.example.com/api/dex/callback
-              clientID: XXXXXXXXXXXXX.apps.googleusercontent.com
-              clientSecret: XXXXXXXXXXXXX
-              serviceAccountFilePath: /tmp/oidc/googleAuth.json
-              adminEmail: admin-email@example.com
-            type: google
-            id: google
-            name: Google
+3. Edit `argocd-cm` and add the following `url` and `dex.config` to the data section, replacing `clientID` and `clientSecret` with the values you saved before, `adminEmail` with the address for the admin user you're going to impersonate, and editing `redirectURI` with your Argo CD domain (note that the `type` is now `google` instead of `oidc`):
```

Comment:
```suggestion
3. Edit `argocd-cm` and add the following `url` and `dex.config` to the data section, replacing `clientID` and `clientSecret` with the values you saved before, `adminEmail` with the address for the admin user you're going to impersonate, and editing `redirectURI` with your Argo CD domain (note that the `type` is now `google` instead of `oidc`):
```

This has been mentioned already in the docs multiple times. For example

- https://argo-cd.readthedocs.io/en/stable/operator-manual/user-management/google/#configure-argo-to-use-the-new-google-saml-app
- https://argo-cd.readthedocs.io/en/stable/operator-manual/user-management/#oidc-configuration-with-dex

However, if you still want to include this, please see how is the doc already written.